src/csharp/CodeWarsKatas/Algorithms/Week1/Sum-of-odd-numbers-7-kyu/SumOfOddNumbersImpl.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace CodeWarsKatas.Algorithms.Week1.Sum_of_odd_numbers_7_kyu
+{
+    internal class SumOfOddNumbersImpl
+    {
+        public static int Solve(string s)
+        {
+            return new Regex(@"\d+")
+                .Matches(s)
+                .Cast<Match>()
+                .OrderByDescending(x => x.Length)
+                .Select(x => int.Parse(x.Value))
+                .Max();
+        } 
+    }
+}
```

Comment:
Can we include the examples as comments?
Like:

-----------
Example:
             1
          3     5
       7     9    11
   13    15    17    19
21    23    25    27    29
...
Calculate the row sums of this triangle from the row index (starting at index 1) e.g.:

row_sum_odd_numbers(1); # 1
row_sum_odd_numbers(2); # 3 + 5 = 8